tools/windows/DatadogAgentInstaller/CustomActions/ProcessUserCustomActions.cs

```diff
@@ -170,9 +171,9 @@ private void ParseUserName(string account, out string userName, out string domai
             // We do not use CredUIParseUserName because it does not handle some cases nicely.
             // e.g. CredUIParseUserName(host.ddev.net\user) returns userName=.ddev.net domain=host.ddev.net
             // e.g. CredUIParseUserName(.\user) returns userName=.\user domain=
-            if (account.Contains("\\"))
+            var sepChars = new[] { '\\', '@' };
```

Comment:
This doesn't account for the inverted `username@domain` order with the `@` sign. I think the tests are passing because this function is only used as a fallback for when the initial `LookupAccountName` fails, but the mock doesn't fail.